core/vm/jump_table.go

```diff
@@ -40,6 +40,8 @@ type operation struct {
 
 	// memorySize returns the memory size required for the operation
 	memorySize memorySizeFunc
+
+	undefined bool
```

Comment:
This is `undefined` rather than `defined` because this way it can be initialized only in one place (bottom of this file) only for undefined instructions (i.e. defined is default)

Reply:
Another idea to achieve this is to have a separate bitmap for defined opcodes and return it together with jump table, i.e.
```go
type JumpTable struct {
	operations [256]*operation
	defined    [32]byte
}
```